### ISSUE TITLE
Doc Accessibility ACC-010: Schema missing alternative text (moderate)

### ISSUE THREAD

**Issue by tfrancart** [sparna-git/shacl-play#328]
- add `role="img"` to svg diagrams
- add `<title>` element : "Diagram for shape xxxxxx"
- add `<description>` element : "This diagram shows a semantic web data model centered around the eli:Work entity"
- ajout de l'attribut `aria-labelledby="schema-title schema-desc"` sur le SVG

**Comment by tfrancart** [comment]
Modifications à faire dans le code Java, pas dans la XSL.
